client/src/app/site/agenda/components/topic-detail/topic-detail.component.html

```diff
@@ -0,0 +1,80 @@
+<os-head-bar [nav]="false" [backButton]=true [allowEdit]="true" [editMode]="editTopic" (editEvent)="setEditMode($event)"
+    (saveEvent)="saveTopic()">
+
+    <!-- Title -->
+    <div class="title-slot">
+        <h2>
+            <span *ngIf="newTopic && !editTopic" translate>New</span>
+            <span *ngIf="editTopic && !newTopic" translate>Edit</span>
+            <!-- Whitespace between "New" and "Topic" -->
+            <span>&nbsp;</span>
+            <span translate>Topic</span>
+        </h2>
+    </div>
+
+    <!-- Menu -->
+    <div class="menu-slot">
+        <button type="button" mat-icon-button [matMenuTriggerFor]="topicExtraMenu">
+            <mat-icon>more_vert</mat-icon>
+        </button>
+    </div>
+
+</os-head-bar>
+
+<div *ngIf="topic" class="topic-container on-transition-fade">
+
+    <div class="topic-title">
+        <h2 *ngIf=!editTopic>{{ topic.title }}</h2>
+        <h2 *ngIf=editTopic>{{ topicForm.get("title").value }}</h2>
+    </div>
+
+    <mat-card *ngIf="topic || editTopic" class="topic-text">
+        <div>
+            <span *ngIf=!editTopic>
+                {{topic.text}}
+            </span>
```

Comment:
Make some gray notice, if there is no text, so there isn't a empty white box. Maybe the same as in statute paragraphs, motion comments or motion comment sections.